server/settings.py

```diff
@@ -33,6 +33,8 @@
 SITE_URL = env("SERVER_URI", default="http://localhost:8000/")
 SITE_HOSTNAME = furl(SITE_URL).host
 ALLOWED_HOSTS = [SITE_HOSTNAME]
+APP_VERSION_STRING = env.str('APP_VERSION_STRING')
+EVAL_FRAME_ORIGIN = env.str('EVAL_FRAME_ORIGIN')
```

Comment:
Let's set some defaults of these, this will allow the `collectstatic` command run in the Dockerfile to work. I would recommend `dev` for APP_VERSION_STRING and maybe `None` for EVAL_FRAME_ORIGIN?